spec/support/cache.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+RSpec.configure do |config|
+  config.after { Rails.cache.clear }
```

Comment:
because the cache is disabled in the test env by default, I think that you can move the `after { Rails.cache.clear }` into the `context "with cache" do` of the `payment_method_spec.rb`. It's the only spec that uses the cache, right?